src/resources/postcss/components/_search.pcss

```diff
@@ -68,26 +69,82 @@
 			}
 
 			&:first-child {
-				flex: 1 1 34.5%;
+				flex: 1 1 100%;
 			}
 
 			&:nth-child(2) {
-				flex: 1 1 37.5%;
+				flex: 1 1 100%;
 			}
 
-			&:nth-child(3) {
-				flex: 1 1 28%;
+			&:last-child {
+				.tribe-common-c-search__input {
+					@media (--viewport-medium) {
+						border-right: 0;
+					}
+				}
+
 			}
+
 		}
 	}
 
 	.tribe-common-c-search__input {
+
 		@media (--viewport-medium) {
-			border: none;
 			border-right: 1px solid var(--color-border-default);
 			height: auto;
 			padding: 7px 18px 8px;
 		}
+
+		&:focus {
```

Comment:
this probably belongs to a more global `input` style rather than specific to search. let's move this to `src/resources/postcss/base/forms/_text.pcss`